### PR TITLE
ortest: added the lua-tablepool, lua-resty-signal, and lua-resty-shell libraries.

### PR DESCRIPTION
**Pull request by thibaultcha**

### CODE REVIEW

**Comment by agentzh** [comment]
@thibaultcha You also need to update the `index.tt` file as well as the `gen-user-data` and `gen-report` files to add these components. It's also mentioned in the ec2-test-cluster document I wrote earlier.

**Comment by thibaultcha** [comment]
@agentzh My bad, updated now.

File: misc/logs/gen-user-data [hunk 1]
```diff
@@ -21,7 +21,7 @@ my %cpu_lualibs = (
 );
 
 my @clibs = qw(
-    sregex resty-cli
+    sregex resty-cli lua-resty-signal lua-resty-shell
```
Comment (agentzh) [inline misc/logs/gen-user-data]: This is not right. They are not pure C libraries. They are Lua libraries. The `lua-resty-signal` and `lua-tablepool` should goto `%cpu_lualibs` instead while `lua-resty-shell` should goto `@lualibs` instead.

**Review by agentzh — CHANGES_REQUESTED** [review]

File: misc/logs/gen-user-data (hunk 1 above)
Comment (thibaultcha) [reply misc/logs/gen-user-data]: Ok, since both have C components I assume they'd go here for the `tv` mode, but will move them.

Comment (thibaultcha) [reply misc/logs/gen-user-data]: Updated.

**Review by agentzh — APPROVED** [review]

**Pull request merged**